src/features/Search/components/WideCard/WideCard.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { memo, VFC } from 'react';
+import { Box, Image, Link, Tooltip } from '@chakra-ui/react';
+import { Link as RouterLink } from 'react-router-dom';
+import { Platform } from '../../../../models/platform';
+import { Language } from '../../../../models/language';
+import { Icon } from '../../../../components/Icon/Icon';
+import { VisualNovel } from '../../../../models/visualNovels/visualNovel';
+import { CardDetail } from '../CardDetail/CardDetail';
+import { CardDetailList } from '../CardDetailList/CardDetailList';
+
+interface Props {
+
+  /** Visual novel data. */
+  readonly vn: VisualNovel;
```

Comment:
So this card is vn-specific only?

Reply:
For now yes, I will decompose them when needed

Reply:
Ok, but don't resolve such conversations by yourself, because reviewers can miss your answer